README.md

```diff
@@ -14,30 +14,47 @@ detailed [API documentation](http://www.opesci.org/devito).
 
 ## Quickstart
 
-Devito can be installed from GitHub via pip:
+The recommended way to install Devito uses the Conda package manager
+for installation of the necessary software dependencies. Please
+install either [Anaconda](https://www.continuum.io/downloads) or
+[Miniconda](https://conda.io/miniconda.html) using the instructions
+provided at the download links.
+
+To install the editable development version Devito, including examples,
+tests and tutorial notebooks, please run the following commands:
 ```
-pip install --user git+https://github.com/opesci/devito.git
+git clone https://github.com/opesci/devito.git
+cd devito
+conda env create -f environment.yml
+source activate devito
+pip install -e .
```

Comment:
How difficult is it to automate the Conda installation and these instructions in an ``install_devito.sh`` script?